### PR TITLE
Bump python version

### PR DESCRIPTION
**Pull request by yv-svstl**
drop 3.8 support
add 3.11, 3.12 support
bump confluent-kafka

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -10,11 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        pydantic-settings-version: [ '~=2.0.0', '' ]
+        pydantic-settings-version: [ '~=2.0.0', '']
         # ToDo (tribunsky.kir): make avro extra too but do not forget to require requests in the sake of SRs
-        dataclasses-avroschema-version: [ '==0.53.0', '~=0.53.0', '==0.54.0', '' ]
-        # 3.11 has no wheel build: https://pypi.org/project/confluent-kafka/1.9.2/#files
-        python-version: [ '3.8', '3.9', '3.10']
+        dataclasses-avroschema-version: [ '==0.53.0', '~=0.53.0', '==0.54.0', '']
```
Comment (tofuurem) [inline .github/workflows/pull-request.yml:15]: I would add version 0.60 to the tests immediately and fix the bug that will come out

File: setup.cfg [hunk 2]
```diff
@@ -27,10 +27,10 @@ no-binary = confluent-kafka
 
 [options]
 packages = find:
-python_requires = >=3.8
+python_requires = >=3.9
 include_package_data = True
 install_requires =
-    confluent-kafka[avro]>=1.2.0
+    confluent-kafka[avro]>=2.5.3
```
Comment (tofuurem) [inline setup.cfg:33]: 2.5.3? maybe 2.4.0 or earlier 1.9.0/2.0.0

Comment (yv-svstl) [reply setup.cfg:33]: Any reason why?

Comment (tofuurem) [reply setup.cfg:33]: We use 2.4.0 for example -_-

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (yv-svstl) [reply .github/workflows/pull-request.yml]: Don't see any

**Review by tofuurem — APPROVED** [review]

**Pull request merged**